### PR TITLE
Updated contributors in main readme.md

### PR DESCRIPTION
**Pull request by sb-decoder**
Fixed issue: #173

### CODE REVIEW

**Comment by sb-decoder** [comment]
@Verisimilitude11 please check

**Comment by Verisimilitude11** [comment]
> @Verisimilitude11 please check

Looks great! Could you actually please move this information to a file in the docs folder and link it in the README? The content looks great!

**Review by Verisimilitude11 — CHANGES_REQUESTED** [review]
Looks great! Could you actually please move this information to a file in the docs folder and link it in the README? The content looks great! @sb-decoder

File: docs/Contribution Guideline/Contribution_Guideline.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Contribution Guideline :
```
Comment (Verisimilitude11) [inline docs/Contribution Guideline/Contribution_Guideline.md:1]: Contribution Guidelines:

**Comment by sb-decoder** [comment]
@Verisimilitude11 Done!

**Review by Verisimilitude11 — CHANGES_REQUESTED** [review]
Rename to Contribution Guidelines. I'll approve it once you make this change! Thanks for working on this!

**Review by Verisimilitude11 — APPROVED** [review]

**Comment by Verisimilitude11** [comment]
@sb-decoder I'd really appreciate it you would give this repository a star to support our mission of making DNA analysis tools available to all!

**Comment by sb-decoder** [comment]
Yeah! starred .can you please add hacktoberfest-accepted lebel

**Comment by Verisimilitude11** [comment]
> Yeah! starred .can you please add hacktoberfest-accepted lebel

Thanks for your support! Looking forward to working with you in the future.

**Pull request merged**